### PR TITLE
Optimize code

### PR DESCRIPTION
**Pull request by tanmay3011**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://tanmay3011-optimize-code.spree-spree-9ed724b77adec04c49c2.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://tanmay3011-optimize-code.spree-spree-9ed724b77adec04c49c2.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: core/app/models/concerns/spree/adjustment_source.rb [hunk 1]
```diff
@@ -12,11 +12,11 @@ module AdjustmentSource
     def create_adjustment(order, adjustable, included = nil)
       amount = compute_amount(adjustable)
       return if amount == 0
-      adjustments.new(order: order,
+      adjustments.create(order: order,
                       adjustable: adjustable,
                       label: label,
                       amount: amount,
-                      included: included).save
+                      included: included)
```
Comment (JDutil) [inline core/app/models/concerns/spree/adjustment_source.rb]: Please update this.

**Comment by JDutil** [comment]
This appears to break specs as they don't appear to be the typical frail backend specs there are failures on core as well.

**Comment by tanmay3011** [comment]
Okay! Will look into it.

**Comment by tanmay3011** [comment]
@JDutil Fixed!

**Comment by JDutil** [comment]
Thanks

**Pull request merged**